models/stg_hubspot__deal.yml

```
@@ -161,3 +161,18 @@ models:
         description: '{{ doc("history_name") }}'
       - name: new_value
         description: '{{ doc("history_value") }}'
+
+  - name: stg_hubspot__merged_deal
+    description: Each record represents a deal that has been merged along with the related deal it's been merged into.
+    columns:
+      - name: merged_deal_id
+        description: ID of deal that has been merged.
+        tests:
+          - unique
+          - not_null
+      - name: deal_id
+        description: The ID of the deal the record has been merged INTO.
```

Is this true? I feel there could be scenarios where this is not in fact unique. I think the following would be a more representative test for this model.
```suggestion
      - name: merged_deal_id
        description: ID of deal that has been merged.
        tests:
          - not_null
      - name: deal_id
        description: The ID of the deal the record has been merged INTO.
        tests:
          - not_null
```

---

@fivetran-joemarkiewicz non-unique scenarios as in where a merged deal could have been merged into more than 1 deal and therefore show up more than once in the merged_deal table? Updating the tests make sense but just wondering if you had any specifics that popped into mind